web_tool_script_3.R

```diff
@@ -163,6 +163,11 @@ js_translations <- jsonlite::fromJSON(
   txt = path(template_path, "data/translation/js_labels.json")
 )
 
+sector_order <- readr::read_csv(
+  path(template_path, "data/sector_order/sector_order.csv"),
```

Comment:
Best to use `path("to", "somewhere")` instead of `path("to/somewhere")`. Different platforms use different path separators -- avoid "/".

Reply:
Our tests on GitHub actions show [this fails](https://github.com/2DegreesInvesting/PACTA_analysis/runs/1523994553?check_suite_focus=true). As you disclose, you can indeed wait for https://github.com/2DegreesInvesting/create_interactive_report/pull/313 or you may rewrite this in such a way that you keep the ecosystem working at all times. For example, you may now add something like 

```r
if (fs::file_exists("...") {
  # read file
} else {
  # do nothing
}
```

Reply:
You will have to come back after https://github.com/2DegreesInvesting/create_interactive_report/pull/313 and remove this little logic, but at lease you move in the direction you want, and always in the green test-zone.

Reply:
> Best to use `path("to", "somewhere")` instead of `path("to/somewhere")`. Different platforms use different path separators -- avoid "/".

I'm not very familiar with the usage of the `path()` function but it sounds reasonable - will modify according to your suggestion.

Reply:
@maurolepore see 5e13306